src/module/Calculator.test.js

```diff
@@ -0,0 +1,12 @@
+/** @format */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+import Calculate from './Calculator';
+
+describe('Calculator page component <Calculator />', () => {
+  test('Snapshot of the home page', () => {
+    const tree = renderer.create(<Calculate />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
![image](https://github.com/derrick1451/math-magician/assets/93039359/da00bf79-47f3-4e3f-b1cb-d007be196a93)
- [ ] It would be nice if you follow the requirements which clearly say what you have to stimulate the user interactions such as clicking the buttons and changing the value on the calculator screen 🤔 Kindly add the testing for user interactions with your app to follow the project requirements and test your app better 👍

Hint: You can search about the **fire event testing in react** etc to get an idea 👍